### DISCUSSION TITLE
Upgrade operator from 3.6 to 3.7

### DISCUSSION THREAD

**Discussion by dkoci** [quay/quay#1574] [General]
Hi.
My current operator version is 3.6 (got it on operatorhub.io). I would like to upgrade it to version 3.7. Is there any instructions how to do it?
I am running it on OKD.

Thank you

**Comment by dmesser** [comment]
You can check out the documentation: https://docs.projectquay.io/deploy_quay_on_openshift_op_tng.html#operator-upgrade
